addons/web/static/tests/fields/field_utils_tests.js

```diff
@@ -297,7 +298,208 @@ QUnit.test('parse percentage', function(assert) {
+    weekdaysParseExact : true,
+    longDateFormat : {
+        LT : 'HH:mm',
+        LTS : 'HH:mm:ss',
+        L : 'DD/MM/YYYY',
+        LL : 'D MMMM YYYY',
+        LLL : 'D MMMM YYYY HH:mm',
+        LLLL : 'dddd D MMMM YYYY HH:mm'
+    },
+    calendar : {
+        sameDay : '[Aujourd’hui à] LT',
+        nextDay : '[Demain à] LT',
+        nextWeek : 'dddd [à] LT',
+        lastDay : '[Hier à] LT',
+        lastWeek : 'dddd [dernier à] LT',
+        sameElse : 'L'
+    },
+    relativeTime : {
+        future : 'dans %s',
+        past : 'il y a %s',
+        s : 'quelques secondes',
+        ss : '%d secondes',
+        m : 'une minute',
+        mm : '%d minutes',
+        h : 'une heure',
+        hh : '%d heures',
+        d : 'un jour',
+        dd : '%d jours',
+        M : 'un mois',
+        MM : '%d mois',
+        y : 'un an',
+        yy : '%d ans'
+    },
+    dayOfMonthOrdinalParse: /\d{1,2}(er|)/,
+    ordinal : function (number, period) {
+        switch (period) {
+            case 'D':
+                return number + (number === 1 ? 'er' : '');
+            default:
+            case 'M':
+            case 'Q':
+            case 'DDD':
+            case 'd':
+                return number + (number === 1 ? 'er' : 'e');
+            case 'w':
+            case 'W':
+                return number + (number === 1 ? 're' : 'e');
+        }
+    },
+    week : {
+        dow : 1, // Monday is the first day of the week.
+        doy : 4  // The week that contains Jan 4th is the first week of the year.
+    }
+});
+
+QUnit.test('parse datetime', function(assert) {
+    assert.expect(6);
+
+    var dateStr, date1, date2;
+
+    // English
+    moment.locale('en');
+    _.extend(core._t.database.parameters, {date_format: '%m/%d/%Y', time_format: '%H:%M:%S'});
+    assert.throws(function() {fieldUtils.parse.datetime("13/01/2019 12:00:00", {}, {})}, /is not a correct/, "Wrongly formated dates should be invalids");
+    assert.throws(function() {fieldUtils.parse.datetime("1899-01-01 12:00:00", {}, {})}, /is not a correct/, "Dates before 1900 should be invalids");
+
+    dateStr = '01/13/2019 10:05:45';
+    date1 = fieldUtils.parse.datetime(dateStr);
+    date2 = moment.utc(dateStr, ['MM/DD/YYYY HH:mm:ss'], true);
+    assert.equal(date1.format(), date2.format(), "Date with leading 0");
+
+    dateStr = '1/14/2019 10:5:45';
+    date1 = fieldUtils.parse.datetime(dateStr);
+    date2 = moment.utc(dateStr, ['M/D/YYYY H:m:s'], true);
+    assert.equal(date1.format(), date2.format(), "Date without leading 0");
+
+    // French
+    moment.locale('fr');
+    _.extend(core._t.database.parameters, {date_format: '%d/%m/%Y', time_format: '%H:%M:%S'});
+    dateStr = '15/01/2019 10:05:45';
+    date1 = fieldUtils.parse.datetime(dateStr);
+    date2 = moment.utc(dateStr, ['DD/MM/YYYY HH:mm:ss'], true);
+    assert.equal(date1.format(), date2.format(), "Month/day inverted");
+
+    // Norwegian
+    moment.locale('nb');
```

Comment:
i think that testing with one locale (!= en) is enough here

Reply:
Norwegian is interesting because month names are different from english, I would get rid of french instead.